data/config/presale.mainnet.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
   wrapper: {
     owner:            '0x5e8c17a6065c35b172b10e80493d2266e2947df4',       // Agent of AN DAO
     registry:         '0xF9dDa954adf5E54b89f988C1560553A0a387CCE1',       // Court's Jurors Registry instance on Mainnet
-    presale:          '0xf89c8752d82972f94a4D1331e010eD6593e8EC49'        // Fundraising presale instance on Mainnet
+    presale:          '0xf89c8752d82972f94a4D1331e010eD6593e8EC49',       // Fundraising presale instance on Mainnet
```

Comment:
This was while merging that other PR, right? I have this comma locally.

Reply:
Ahh, it may have been because of resolving the merge conflict in #39.